kubestash/__init__.py

```diff
@@ -313,6 +314,10 @@ def kube_replace_secret(args, namespace, secret, data):
     current_secret = kube.read_namespaced_secret(secret, namespace)
     current_secret_data = current_secret.data
 
+    for secretkey in current_secret_data:
+        if secretkey not in data:
```

Comment:
@telnet-harsha Can we add a log line here ?

Reply:
okay